protocol.md

```diff
@@ -870,6 +902,12 @@ export interface TextDocumentClientCapabilities {
 ```typescript
 interface ClientCapabilities {
 	/**
+	 * The client supports receiving the result solely through $/partialResult notifications for requests from the client to the server.
+	 * If true, the server should send all data through $/partialResult and send null as the result in the final response.
```

Comment:
Yeah I have seen that explicit ordering in the lsp doc before. I don't know why that is in the spec, it should rather just have the second half about the order is important w.r.t. the correctness of results. We explicitly ignore it since we have a multi-user environment and someone doing a slow reference operation shouldn't block other people doing a hover.